combat-trainer.lic

```diff
@@ -2248,8 +2254,16 @@ class AbilityProcess
 
     if battle_cry_data
       echo "Selected battle cry: #{battle_cry_data}" if $debug_mode_ct
+      echo "Using roar helm: #{@roar_helm}, with roar helm noun: #{@roar_helm_noun}" if $debug_mode_ct
+
       command = battle_cry_data['command']
       command = (command + " at " + battle_cry_data['target_enemy']) if battle_cry_data['target_enemy']
+
+      # Use roar helm if specified
+      fput("scream #{@roar_helm_noun}") if @roar_helm
```

Comment:
Can we avoid two new settings and have this "if" check based on `roar_helm` nil or not?

Reply:
I believe we need to know the noun of the helm in case `helm` isn't it. We could use one setting, perhaps, called `roar_helm_noun`, and just check if that's not empty?

Reply:
Or are you thinking to grab the noun from our currently worn helm somehow, maybe via EquipManager?

Reply:
PS - There is also a more complex version of implementing roar helms that involves something like your idea: adding an `:is_roar: true` element to gear list, then making sure we're wearing that helm before we try the `scream helm`, which would prevent failed screams when cycling armor and the helm isn't being worn.

I don't 100% know how to do that and would need to do a bit of study. Thoughts?